src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -94,9 +95,22 @@ class ContainerBuilder extends Container implements TaggedContainerInterface
      */
     private $usedTags = array();
 
+    private $context;
+
     private $compiled = false;
 
     /**
+     * @param ParameterBagInterface|null $parameterBag A ParameterBagInterface instance
+     * @param Context|null               $context      A Context instance, or null
```

Comment:
erm, I still fail to see what is in the phpdoc that is not in the signature, new parameter or not. Or I am missing something?